.github/workflows/build.yml

```diff
@@ -8,46 +8,44 @@ on:
 
 jobs:
   build:
-    runs-on: ${{ matrix.os }}
+    name: "build"
+    runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
         java: [ 8, 11 ]
-        os: [ ubuntu-18.04 ]
     steps:
       # step 1
       - name: "Checkout"
-        uses: actions/checkout@v2.3.4
+        uses: actions/checkout@v2.4.0
 
       # step 2
       - name: "Set up Java JDK"
-        uses: actions/setup-java@v2.1.0
+        uses: actions/setup-java@v2.5.0
         with:
           distribution: 'zulu'
           java-version: ${{ matrix.java }}
 
       # step 3
       - name: "Set up ENV"
         # https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable
-        run: if [ "${{ matrix.java }}" == "8" ]; then
-               echo "IMAGE_NAME=openjdk:8u212-jre-alpine" >> $GITHUB_ENV;
-             elif [ "${{ matrix.java }}" == "11" ]; then
-               echo "IMAGE_NAME=openjdk:11-jre-stretch"   >> $GITHUB_ENV;
-             fi
+        run: |
+          if [ "${{ matrix.java }}" == "8" ]; then
+            echo "IMAGE_NAME=openjdk:8u212-jre-alpine" >> $GITHUB_ENV;
+          elif [ "${{ matrix.java }}" == "11" ]; then
+            echo "IMAGE_NAME=openjdk:11-jre-stretch"   >> $GITHUB_ENV;
+          fi
 
       # step 4
       - name: "Build with Maven"
-        env:
-          REGISTRY_USERNAME: ${{ secrets.REGISTRY_USERNAME }}
-          REGISTRY_PASSWORD: ${{ secrets.REGISTRY_PASSWORD }}
-        # https://docs.github.com/cn/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#github-context
-        run: if [ "${{matrix.java}}" == "8" ] && [ "${{github.event_name}}" == "push" ] && [ "${{github.ref}}" == "refs/heads/develop" ]; then
-               ./mvnw -T 4C clean test -e -Pimage -DskipTests=false -Dcheckstyle.skip=false -Dlicense.skip=false -B -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn;
-             else
-               ./mvnw -T 4C clean test -e         -DskipTests=false -Dcheckstyle.skip=false -Dlicense.skip=false -B -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn;
-             fi
+        run: |
+          if [ "${{ matrix.java }}" == "8" ]; then
+            ./mvnw -T 4C clean test -Dcheckstyle.skip=false -Dlicense.skip=false -e -B -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn;
+          elif [ "${{ matrix.java }}" == "11" ]; then
```

Comment:
No essential differences are found between JDK8 and JDK11.

Reply:
JDK11：`checkstyle.skip` and `license.skip` is `true`.